### ISSUE TITLE
oce.plot.ts() can spew warnings (for some oceglider data)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2241]
See https://github.com/dankelley/oceglider/issues/118 for a bug report that went away when I restarted RStudio.

I think that is because the plot window size was changed.  Some grepping suggests that the error is at the lie below (and the one after that).  I will look into why this happens, likely by Saturday.  But I think the problem is most likely due to having a lot of NA values.  I will see if I can fix it in oce and also in oceglider, since oce won't get a CRAN release for half a year or so.

https://github.com/dankelley/oce/blob/f2953ada8d8bdf3459eb7949abee0c405c480e2a/R/oce.R#L1289

**Comment by dankelley** [comment]
I think this is related
```R
> min(c(NA,NA), na.rm=TRUE)
[1] Inf
Warning message:
In min(c(NA, NA), na.rm = TRUE) :
  no non-missing arguments to min; returning Inf
```

**Comment by dankelley** [comment]
It should not be computing `length(...)` but rather the length of the finite values in ...

**Comment by dankelley** [comment]
Here's a test file, and screenshot showing the traceback.
```R
library(oce)
N <- 50000 # make long to force oce.plot.ts() to simplify it
y <- sin(seq_len(N) * 2 * pi / 5000)
t <- as.POSIXct(Sys.Date()) + 1:N
y[1000:5000] <- NA
options(warn = 3)
oce.plot.ts(t, y)
```

<img width="1266" alt="Screenshot 2024-10-06 at 10 00 26 AM" src="https://github.com/user-attachments/assets/fcc62d5a-5d3d-40ac-9e80-3cae86ca53db">

**Comment by dankelley** [comment]
I've solved the problem in commit ef00e92009cdc13f4e36d60c35fc82a79397fc85 of "develop".

I'll leave this issue open until remote tests have passed.

**Comment by dankelley** [comment]
Forgot to check on the remote checks.  They worked, so I'm closing this.
